internal/envoy/bootstrap.go

```diff
@@ -49,6 +49,13 @@ func Bootstrap(c *BootstrapConfig) *bootstrap.Bootstrap {
 						SocketAddress(c.xdsAddress(), c.xdsGRPCPort()),
 					),
 				},
+				UpstreamConnectionOptions: &api.UpstreamConnectionOptions{
+					TcpKeepalive: &envoy_api_v2_core.TcpKeepalive{
+						KeepaliveProbes:   protobuf.UInt32(3),
```

Comment:
Good to see envoy continues its (in)consistent use of durations, timeouts in seconds, and various unsigned integers